{{cookiecutter.project_slug}}/frontend/vite.config.ts

```diff
@@ -1,12 +1,25 @@
-import { defineConfig } from 'vite'
+import { defineConfig, loadEnv } from 'vite'
 import react from '@vitejs/plugin-react'
 import tsconfigPaths from 'vite-tsconfig-paths'
 
-// https://vitejs.dev/config/
-export default defineConfig({
-	plugins: [react(), tsconfigPaths()],
-	server: {
-		host: '0.0.0.0',
-		port: 3000
-	}
-})
+
+export default ({ mode }) => {
+  process.env = { ...process.env, ...loadEnv(mode, process.cwd(), '') }
+
+  return defineConfig({
+    plugins: [
+      react(),
+      tsconfigPaths(),
+    ],
+    server: {
+			host: '0.0.0.0',
```

Comment:
**nitpick** tabs instead of spaces

Reply:
Not very sure why github displays like this but it is in correct place. We have prettier setup and it automatically fixing it with precommit

Reply:
@sedatbasar I don't think prettier was ran on the cookie-cutter frontend source because I see tabs instead of spaces in all the `ts` and `tsx` files. I completely missed this when reviewing the PR to add React, good catch @rennerocha!

Reply:
Ohh i see it. Yes prettier is working but it is using `useTabs: true` :) just removed that